pkg/module/remote.go

```diff
@@ -103,6 +104,19 @@ func (r *Remote) getCredentials(ctx cpi.Context) credentials.Credentials {
 	return creds
 }
 
+// See: github.com/open-component-model/ocm/pkg/contexts/credentials/repositories/dockerconfig/repository.go#IsEmptyAuthConfig()
+func isEmptyAuth(creds credentials.Credentials) bool {
+
+	if len(creds.GetProperty("auth")) != 0 {
+		return false
+	}
+	if len(creds.GetProperty("username")) != 0 {
```

Comment:
Just wondering if it could happen that `username` property is empty while `auth` is not? Do you think it makes sense to rework this check into returning true if `auth` or `username` is empty rather than returning early if only `auth` is not? WDYT?